core/src/test/scala/org/apache/spark/sql/delta/DelegatingLogStoreSuite.scala

```diff
@@ -120,6 +120,13 @@ class DelegatingLogStoreSuite
         expClassName = DelegatingLogStore.defaultAzureLogStoreClassName,
         expAdaptor = true)
     }
+    for (scheme <- DelegatingLogStore.gsSchemes) {
+      testDelegatingLogStore(
+        scheme,
+        schemeConf = None,
+        expClassName = DelegatingLogStore.defaultGCSLogStoreClassName,
+        expAdaptor = true)
+    }
```

Comment:
please update the test below "DelegatingLogStore resolution using customized scheme confs"".
- add `"io.delta.storage.GCSLogStore"` and `classOf[org.apache.spark.sql.delta.storage.GCSLogStore].getName,`